### PR TITLE
schema: revise description of `<catRel>`

### PR DESCRIPTION
**Pull request by pe-ro**

### CODE REVIEW

File: source/modules/MEI.header.xml [hunk 1]
```diff
@@ -681,7 +681,8 @@
   </elementSpec>
   <elementSpec ident="catRel" module="MEI.header">
     <gloss versionDate="2022-05-18" xml:lang="en">category relationship</gloss>
-    <desc xml:lang="en">Contains the name of a related category.</desc>
+    <desc xml:lang="en">(category relationship) Contains the name; i.e., label, of a related
```
Comment (ahankinson) [inline source/modules/MEI.header.xml]: Doesn't this repeat what is in the `gloss` element above?

Comment (pe-ro) [reply source/modules/MEI.header.xml]: You're right, it does.  Another cut-and-paste error.  The parenthetical part of `<desc>` should be removed.

Comment (ahankinson) [inline source/modules/MEI.header.xml]:
```suggestion
    <desc xml:lang="en">Contains the name; i.e., label, of a related
```

Comment (bwbohl) [reply source/modules/MEI.header.xml]:
```suggestion
    <desc xml:lang="en">Contains the name, i.e., label, of a related
```

**Review by musicEnfanthen — APPROVED** [review]
Thank you.

**Review by bwbohl — APPROVED** [review]

**Pull request merged**